draft-ietf-tls-dtls13.md

```diff
@@ -1275,19 +1294,15 @@ Client                                            Server
 ~~~
 Client                                            Server
 
-                                                          +----------+
-                       <--------       [NewSessionTicket] | Flight 1 |
-                                                          +----------+
+                                                          +--------+
+                       <--------       [NewSessionTicket] | Flight |
+                                                          +--------+
 
-                                                          +----------+
-[ACK]                  -------->                          | Flight 2 |
-                                                          +----------+
+                                                          +--------+
+[ACK]                  -------->                          | Flight |
+                                                          +--------+
 ~~~
-{: #dtls-post-handshake-ticket title="Message flights for the new session ticket message"}
-
-Note: The application data sent by the client is not included in the
-timeout and retransmission calculation.
-
+{: #dtls-post-handshake-ticket title="Message flights for the NewSessionTicket message"}
```

Comment:
Do we also want to mention KeyUpdate/NewConnectionID/RequestConnectionID here?  It seems overkill to have a separate figure showing the message and ACK for each of them...

Reply:
Yeah, I guess that would work.